web/app/healthcheck/healthcheck_test.go

```diff
@@ -0,0 +1,28 @@
+package healthcheck
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+)
+
+func TestHandler(t *testing.T) {
+	w := httptest.NewRecorder()
```

Comment:
For something like a healthcheck I'd want to test that e2e to get as much confidence as possible. This'll also test middleware, routing, and serialisiation/deserialization is setup correctly.
My understanding is that golang attempts to make this easy, but this is where my ignorance shows: I've not done it yet :D 
Looks like there's an example of how we can extract the code that sets up the handlers from the actual server implementing those handlers in this link: https://bignerdranch.com/blog/using-the-httptest-package-in-golang/
(the bit with `TestPseudOAuth`)  

That can come later though.